src/Goerli-DssSpell.sol

```diff
@@ -31,47 +31,34 @@ contract DssSpellAction is DssAction {
     // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/TODO -q -O - 2>/dev/null)"
     string public constant override description = "Goerli Spell";
 
-    // Gelato Keeper Testing Contracts
-    address constant TOP_UP1 = 0xB77827FabA70185D1EaD053648f665d2C41f7906;
-    address constant TOP_UP2 = 0xbe77Cd403Be3F2C7EEBC3427360D3f9e5d528F43;
-
-    // Math
-    uint256 constant MILLION  = 10 ** 6;
-
     // Turn off office hours
     function officeHours() public override returns (bool) {
-        return false;
+        return true;
```

Comment:
Do we need `officeHours` On on Goerli?

Reply:
Going to keep it in place, I think the GSM is only 5 minutes or so.  Let me make sure in goerli though.

Reply:
60 seconds: https://goerli.etherscan.io/address/0xefcd235B1f13e7fC5eab1d05C910d3c390b3439F#readContract

Reply:
The pause delay is set to 60 seconds in Goerli.